IdentityCore/src/MSIDExternalAADCacheSeeder.m

```diff
@@ -69,9 +73,17 @@ - (void)seedTokenResponse:(MSIDTokenResponse *)originalTokenResponse
     
     MSID_LOG_WITH_CTX(MSIDLogLevelInfo, requestParameters, @"Beginning external cache seeding.");
     
-    void (^completionBlockWrapper)(void) = ^
+    [[MSIDTelemetry sharedInstance] startEvent:requestParameters.telemetryRequestId eventName:MSID_TELEMETRY_EVENT_EXTERNAL_CACHE_SEEDING];
+    
+    void (^completionBlockWrapper)(BOOL success) = ^(BOOL success)
     {
         MSID_LOG_WITH_CTX(MSIDLogLevelInfo, requestParameters, @"External cache seeding finished.");
+        
```

Comment:
I'd also like to track if we made network request or got id token from cache to catch regressions where we always make 2 network requests.

Reply:
done